reference/generators/make.rst

```diff
@@ -0,0 +1,82 @@
+
++--------------------------------------+----------------------------------------------------------------------+
+| NAME                                 | VALUE                                                                |
++======================================+======================================================================+
+| CONAN_ROOT_<PKG-NAME>                | Absolute path to root package folder                                 |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_SYSROOT_<PKG-NAME>             | System root folder                                                   |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_INC_PATHS_<PKG-NAME>           | Headers folders                                                      |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_LIB_PATHS_<PKG-NAME>           | Library folders                                                      |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_BIN_PATHS_<PKG-NAME>           | Binary folders                                                       |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_BUILD_PATHS_<PKG-NAME>         | Build folders                                                        |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_RES_PATHS_<PKG-NAME>           | Resources folders                                                    |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_LIBS_<PKG-NAME>                | Binary folders                                                       |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_LIBS_<PKG-NAME>                | Library names to link with                                           |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_DEFINES_<PKG-NAME>             | Library definitions                                                  |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_CFLAGS_<PKG-NAME>              | Library C flags                                                      |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_CPPFLAGS_<PKG-NAME>            | Library CXX flags                                                    |
```

Comment:
The variable name `CPPFLAGS` is misleading.  `CPPFLAGS` is actually short for "C Pre-Processor Flags", nothing to do with C++.  In GNU Make, there is a separate implicit variable called `CXXFLAGS` that we don't do anything with in Conan... that would be specifically for the C++ compiler.